student-information-management-application/src/main/java/com/student/information/management/app/model/student/Student.java

```diff
@@ -21,6 +21,7 @@ public class Student {
     private String email;
     private String address;
     private String contactNumber;
+    private String name;
```

Comment:
what is this name field for? you already have last, first and middle name already.